monkestation/code/modules/antagonists/wizard/equipment/spellbook.dm

```diff
@@ -0,0 +1,53 @@
+////////////
+// SPELLS //
+////////////
+
+
+/datum/spellbook_entry/lesser_summonapes
+	name = "Lesser Summon Apes"
+	spell_type = /obj/effect/proc_holder/spell/aoe_turf/conjure/lesser_summonapes
+	category = "Defensive"
+
+/datum/spellbook_entry/mutagenic_pulse
+	name = "Mutagenic Pulse"
+	spell_type = /obj/effect/proc_holder/spell/aoe_turf/mutagenic_pulse
+	cost = 1 // making this cost 1 due to mass produced mutadone effectively countering it
+
+
+///////////
+// ITEMS //
+///////////
+
+
+/datum/spellbook_entry/item/magicspray
+	name = "Magical Chem Sprayer"
+	desc = "A magic chem sprayer that will fill itself with unlimted random chemicals."
+	item_path = /obj/item/reagent_containers/spray/chemsprayer/magical
+	category = "Assistance"
+	cost = 1
+
+
+/////////////
+// RITUALS //
+/////////////
+
+
+/datum/spellbook_entry/summon/message//sends a curse of madness message for free without any effect on the crew
```

Comment:
This should only be usable once, make sure it can't be spammed.

Reply:
It already cant be, thats what the active = TRUE does I am pretty sure